### PR TITLE
docs: Add export example in README

### LINKED ISSUE #343: [docs] Add export format example in the documentation
The export format is not very clear from the documentation or the readme, there should be a simple example for readers to easily understand it.

### PR DESCRIPTION
**Pull request by charlesmindee**
This PR illustrates with an example the export format  in README. Any feedback is welcome!

Closes #343

### CODE REVIEW

**Review by fg-mindee — COMMENTED** [review]
Thanks for the PR !

Two high-level comments:
- maybe it's too much information for the README. Perhaps we sould only keep the JSON export example
- I'm wondering if you shouldn't move this all together to the documentation rather than the README. Something like `For examples & further details about the export format, please refer this section of the documentation`

What do you think?

**Comment by charlesmindee** [comment]
I agree, I can move everything to the docs

**Comment by charlesmindee** [comment]
This is how it renders in the models page of the doc:
image

File: README.md [hunk 1]
```diff
@@ -63,12 +63,15 @@ result.show(doc)
...
+Calling the ocr_predictor returns a `Document` object with a nested structure (with `Page`, `Block`, `Line`, `Word`, `Artefact`). 
+To get a better understanding of our document model, check our [documentation](https://mindee.github.io/doctr/documents.html#document-structure):
+
+You can also export a document to a nested dict format (JSON):
 
 ```python
 json_output = result.export()
 ```
-
+For examples & further details about the export format, please refer [this section](https://mindee.github.io/doctr/models.html#export-model-output) of the documentation
```
Comment (fg-mindee) [inline README.md:74]: small typo "please refer **to** ..."

File: README.md [hunk 2]
```diff
@@ -63,12 +63,15 @@ result.show(doc)
 
 ![DocTR example](https://github.com/mindee/doctr/releases/download/v0.1.1/doctr_example_script.gif)
 
-or export them to JSON format (to get a better understanding of our document model, check our [documentation](https://mindee.github.io/doctr/documents.html#document-structure)):
+Calling the ocr_predictor returns a `Document` object with a nested structure (with `Page`, `Block`, `Line`, `Word`, `Artefact`). 
+To get a better understanding of our document model, check our [documentation](https://mindee.github.io/doctr/documents.html#document-structure):
+
+You can also export a document to a nested dict format (JSON):
```
Comment (fg-mindee) [inline README.md:69]: `You can also export them as a nested dict, more appropriate for JSON format:`

File: README.md [hunk 3]
```diff
@@ -63,12 +63,15 @@ result.show(doc)
 
 ![DocTR example](https://github.com/mindee/doctr/releases/download/v0.1.1/doctr_example_script.gif)
 
-or export them to JSON format (to get a better understanding of our document model, check our [documentation](https://mindee.github.io/doctr/documents.html#document-structure)):
+Calling the ocr_predictor returns a `Document` object with a nested structure (with `Page`, `Block`, `Line`, `Word`, `Artefact`).
```
Comment (fg-mindee) [inline README.md:66]: "The predictors returns a `Document` object ...."

File: docs/source/models.rst [hunk 4]
```diff
@@ -178,6 +178,74 @@ Those architectures involve one stage of text detection, and one stage of text r
 
 .. autofunction:: doctr.models.zoo.ocr_predictor
 
+Export model output
+^^^^^^^^^^^^^^^^^^^^
+
+Calling the ocr_predictor returns a `Document` object with a nested structure (with `Page`, `Block`, `Line`, `Word`, `Artefact`).
```
Comment (fg-mindee) [inline docs/source/models.rst:184]: Same here

File: docs/source/models.rst [hunk 5]
```diff
@@ -178,6 +178,74 @@ Those architectures involve one stage of text detection, and one stage of text r
...
+            Word(value='RECEIPT', confidence=0.99),
+            Word(value='DATE', confidence=0.96),
+          ]
+        )]
+        (artefacts): []
+      )]
+    )]
+  )
+
+You can also export a document to a nested dict format (JSON)::
```
Comment (fg-mindee) [inline docs/source/models.rst:205]: same here

File: docs/source/models.rst [hunk 6]
```diff
@@ -178,6 +178,74 @@ Those architectures involve one stage of text detection, and one stage of text r
 
 .. autofunction:: doctr.models.zoo.ocr_predictor
 
+Export model output
+^^^^^^^^^^^^^^^^^^^^
+
+Calling the ocr_predictor returns a `Document` object with a nested structure (with `Page`, `Block`, `Line`, `Word`, `Artefact`). 
+To get a better understanding of our document model, check our :ref:`document_structure` section
+
+A `Document` can be printed out::
```
Comment (fg-mindee) [inline docs/source/models.rst:187]: "Here is a typical `Document` layout:"

File: docs/source/models.rst [hunk 7]
```diff
@@ -178,6 +178,74 @@ Those architectures involve one stage of text detection, and one stage of text r
...
+        (artefacts): []
+      )]
+    )]
+  )
+
+You can also export a document to a nested dict format (JSON)::
+
+  json_output = result.export()
+
+Then when printing `json_output`::
```
Comment (fg-mindee) [inline docs/source/models.rst:209]: "For reference, here is the JSON export for the same `Document` as above:"

**Review by fg-mindee — COMMENTED** [review]
I added a few improvement suggestions!

**Review by fg-mindee — APPROVED** [review]
Thanks again!

**Pull request merged**